hypertrace-trace-enricher/enriched-span-constants/src/main/java/org/hypertrace/traceenricher/enrichedspan/constants/utils/EnrichedSpanUtils.java

```diff
@@ -152,9 +149,10 @@ public static String getBackendPort(Event event) {
     return getStringAttribute(event, BACKEND_PORT_ATTR);
   }
 
-  public static String getBackendPath(Event event) {
-    return getStringAttribute(event, BACKEND_PATH_ATTR);
-  }
+  public static String getBackendPath(Event event) { return getStringAttribute(event, BACKEND_PATH_ATTR); }
+
+  @Nullable
+  public static String getBackendOperationName(Event event) {return getStringAttribute(event, BACKEND_OPERATION_ATTR); }
```

Comment:
Is the method defined in a single line? Can we have the return statement in the next line?

```
public static String getBackendOperation(Event event) {
  return getStringAttribute(event, BACKEND_OPERATION_ATTR);
}
```